Orange/widgets/model/owlogisticregression.py

```diff
@@ -92,10 +92,10 @@ def get_learner_parameters(self):
 def create_coef_table(classifier):
     i = classifier.intercept
     c = classifier.coefficients
-    if len(classifier.domain.class_var.values) > 2:
-        values = classifier.domain.class_var.values
+    if c.shape[0] > 2:
```

Comment:
Previous code was probably not correct. c.shape[0] is not always equal to len(classifier.domain.class_var.values). That is in the case when we have a domain with more class values than contained in a data table. And of course always in the case when number of that values is 2.